public class JedliksToyCar {

     int distance;
	 int battery = 100;

	public static JedliksToyCar buy() {

		JedliksToyCar newCar = new JedliksToyCar();
		return newCar;
	}

	public String distanceDisplay() {

		return "Driven " + distance + " meters";
	}

	public String batteryDisplay() {

		if (battery<=0) {
			return "Battery empty";
		} 
		
		return "Battery at " + battery + "%";
	}

	public void drive() {
		if (battery >0) {
			distance += 20;
		battery-=1;
		} else {
			batteryDisplay();
		}
    }
	}
